hr_timesheet_rounded/models/project_project.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class ProjectProject(models.Model):
+
+    _inherit = 'project.project'
+
+    timesheet_rounding_granularity = fields.Float(
+        string='Timesheet rounding granularity',
+        default=0.25,  # 15 min
```

Comment:
IMO default value should either be 0 or define a field on `res.company` accessible in the config settings to make it customizable.